practice/flutter/flutter/logic.py

```diff
@@ -0,0 +1,15 @@
+"""flutter Logic."""
+
+from . import models
+
+
+def create_and_save_new_flutt(text):
+    """Creates and saves a new flutt.
+
+    >>> stark = create_and_save_new_flutt('winter is coming')
+    >>> stark.text
+    'winter is coming'
```

Comment:
This is one of those instances I would modify this function for the sole purpose of testing it. You don't check the timestamp because it's dependent on when this test runs, but if you make this function take an optional timestamp argument and pass that to the `Flutt` constructor, then you can test it deterministically. Yes, it doesn't test exactly the behavior the app uses, but it gets closer to testing the behavior.